python/openassetio/pluginSystem/PluginSystemManagerInterfaceFactory.py

```diff
@@ -21,7 +21,7 @@
 
 import os
 
-from ..hostApi.ManagerInterfaceFactoryInterface import ManagerInterfaceFactoryInterface
+from ..hostApi import ManagerInterfaceFactoryInterface
```

Comment:
[consider] Should we also remove the `managers` and `managerRegistered` methods from the `PluginSystemManagerInterfaceFactory`?

Reply:
I knew I forgot something! Removed [here], thanks.

Reply:
Which also allowed me to notices [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/535/commits/139ed3462b44150da49cf857f39940985f2faf92).